### ISSUE TITLE
Installer not work anymore

### ISSUE THREAD

**Issue by Neppumuck** [Pieloaf/Debian-DietPi-Install-Script#1]
Hi, thanks for the Script, but today, i get this issue:
update-initramfs: Generating /boot/initrd.img-5.10.0-20-amd64
./debian-dietpi-install.sh: line 70: /boot/dietpi/func/dietpi-set_swapfile: No such file or directory
mkswap: error: /dev/mapper/vg00-lv00 is mounted; will not make swapspace
swapon: /dev/mapper/vg00-lv00: swapon failed: Device or resource busy

Do you have any Idea?
Greetings

**Comment by Pieloaf** [comment]
oh damn its been forever haha this hasnt been kept up to date. After a quick search it looks like the script was renamed so when the script tries to [wget the prep script ](https://github.com/Pieloaf/Debian-DietPi-Install-Script/blob/main/debian-dietpi-install.sh#L32) it fails and so the rest of the script wont run. The file was renamed so we can just point it to the new installer script but it also changed a lot so will need some time to go over the changes and see if the contents of this script are still relevant

**Comment by Pieloaf** [comment]
This is the current file if you wanna try it manually, if not we can take a look into updateing this https://github.com/MichaIng/DietPi/blob/master/.build/images/dietpi-installer

**Comment by Pieloaf** [comment]
tbh reading the docs it looks like the new installer does a lot of the work here so probably just try that: https://dietpi.com/docs/hardware/#basic-information

**Comment by Pieloaf** [comment]
I updated the script now to use the new URL although haven't tested it at all, so use at your own risk.

**Comment by Neppumuck** [comment]
Hi,
thank you very much! When i get an error, i changed the line an then it works:
wget https://raw.githubusercontent.com/MichaIng/DietPi/master/.build/images/dietpi-installer -O dietpi-installer.sh

When you please change it in your script, i will be a happy man :-)

Greetings
Neppu

**Comment by Pieloaf** [comment]
Already changed it in the script yesterday haha https://github.com/Pieloaf/Debian-DietPi-Install-Script/blob/main/debian-dietpi-install.sh#L32 have you tried running it again? if you got another error what was it?

**Comment by Neppumuck** [comment]
Yes, you will get an html error, when you use `https://github.com/MichaIng/DietPi/blob/master/.build/images/dietpi-installer`.
You download an Website but not the script.
You have to use `https://raw.githubusercontent.com/MichaIng/DietPi/master/.build/images/dietpi-installer`.
Thank You

**Comment by Pieloaf** [comment]
Oh shit yea my bad, missed that when I was editing it. Thanks.
Fixed it now
